python_modules/dagster/dagster_tests/core_tests/execution_tests/test_context.py

```diff
@@ -20,3 +23,24 @@ def foo():
         ctx_op()
 
     assert foo.execute_in_process().success
+
+
+def test_instance_check():
+    # turn off any outer warnings filters, e.g. ignores that are set in pyproject.toml
+    warnings.resetwarnings()
```

Comment:
Alex: do these end up being process global settings that effect a whole test session? wonder if theres a way to save and restore. If BK is passing maybe its not worth worrying about for now

Jamie: hmm i dont think so, but not 100% sure. we do this set of calls in other places in our unit tests, and it seems to be fine